tests/generate_vegeta_targets.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This script is unused, but its an example of what I was using initially to test by hand. I'll remove it now rather than in-a-month-when-we-remember, but if anyone else wants to play with these tests its good to know you _can_ build request streams with jq manipulation.

Reply:
This seems like somethign that is really useful. Would we want to keep something like this in the DevEnv branch?